README.md

```diff
@@ -25,6 +25,17 @@ everything is working properly. Output, checkpoint and formatted checkpoint
 files will be generated in the output directory. A log showing the steps
 performed by the script will be available in the log directory.
 
+### Running on different nodes
+
+If your job needs more time, you can run on a different node. The default node is 'short' 
+which has a 24 h time limit. Switching to 'long' changes the time limit to 2 weeks. A full 
+list of nodes can be found [here](https://hpcrcf.atlassian.net/wiki/spaces/TCP/pages/7285967/Partition+List).
+To run on a different node add the name of the node at the end of the submission command. For 
+example to run on a **long** node run the following command:
+```
+bash auto_slurm/scripts/submit_gaussian.bash long
```

Comment:
Nice! much cleaner than having 2 copies of the same code laying around. 
I'm curious, why would anyone ever submit on the short node? would it make more sense to default to long?
Then your jobs that happen to run for 25 hours don't get cancelled.
Is there typically much less jobs on short, so you don't have to wait in a queue?

Reply:
Yep, the last one.